src/pages/index.tsx

```diff
@@ -1,12 +1,124 @@
+            </li>
+            <li>
+              To reduce our expenses for now, we do not yet trace calls or search for deployment
+              transactions. This means:
+              <ul className='list-disc pl-8'>
+                <li>You must provide the transaction on the verification form.</li>
+                <li>
+                  Contract deployed from arbitrary contracts cannot yet be verified. Only contracts
+                  deployed via a standard <code className='text-sm'>create</code> deployment, Nick
+                  Johnson&apos;s <code className='text-sm'>create2</code>{' '}
+                  <ExternalLink
+                    text='deployer'
+                    href='https://github.com/Arachnid/deterministic-deployment-proxy'
+                  />
+                  , or 0age&apos;s <code className='text-sm'>create2</code>{' '}
+                  <ExternalLink
+                    text='deployer'
+                    href='https://etherscan.io/address/0x0000000000FFe8B47B3e2130213B802212439497'
+                  />{' '}
+                  are supported.
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </>
+      ),
+    },
+    {
+      question: 'I like Cove, how can I help it succeed?',
+      answer: (
+        <>
+          Try{' '}
+          <Link href='/verify' className='hyperlink'>
+            verifying
+          </Link>{' '}
+          a contract! This is a public goods project funded solely by hackathon{' '}
+          <ExternalLink text='prizes' href='https://ethglobal.com/showcase/cove-8ohfk' />, so
+          consider donating to TODO or nominating us for the{' '}
+          <ExternalLink
+            text='upcoming'
+            href='https://optimism.mirror.xyz/oVnEz7LrfeOTC7H6xCXb5dMZ8Rc4dSkD2KfgG5W9cCw'
+          />{' '}
+          Optimism RPGF round
+        </>
+      ),
+    },
+  ];
+
   return (
     <>
       <Head />
-      <h1 className='text-center'>{SITE_DESCRIPTION}</h1>
+
+      {/* Header */}
+      <h1 className='text-accent text-center text-base font-semibold'>
+        <Link href='/verify' className='max-w-prose text-sm font-semibold'>
+          Try Cove →
+        </Link>
+      </h1>
+      <p className='mt-2 text-center text-3xl font-bold tracking-tight sm:text-4xl '>
+        {SITE_DESCRIPTION}
+      </p>
+
+      {/* Content */}
+      <div className='flex flex-1 justify-center pt-10'>
+        <dl className='max-w-prose space-y-6'>
+          {faqs.map((faq) => (
+            <div key={faq.question}>
+              <dt className='text-primary font-semibold leading-7'>{faq.question}</dt>
+              <dd className='text-secondary mt-1 leading-7'>{faq.answer}</dd>
+            </div>
+          ))}
+        </dl>
+      </div>
+      <div className='mt-10 flex flex-1 justify-center'>
+        <Link href='/verify' className='btn-primary max-w-prose'>
+          Try it now →
```

Comment:
Instead of "Try it now" maybe we can say "Test it now" or "Test Cove now" or something similar, to help emphasize it's just for testing, not to gain any benefits for the user.

Reply:
Per element, we'll go with "Test Cove now" for both buttons

Reply:
Done in 41a9e07